scripts/SortSparseMatrix.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python
```

Comment:
Please use the previous file naming style -- `sort_sparse_matrix.py`.  Use `git mv SortSparseMatrix.py sort_sparse_matrix.py`, as it's good form to preserve history.

I appreciate that SortSparseMatrix.py is closer to what some other files in this repo use, but file names short, all-lowercase names with underscores-separated words is [Python convention](https://www.python.org/dev/peps/pep-0008/#package-and-module-names).  We should conform those other files to that style over time.

Reply:
Per our chat, we can defer this renaming until we fix all these file names.